### PR TITLE
Fix/ade 552/fixing issues

### PR DESCRIPTION
**Pull request by robinMcA**
Will wait to see what sonar thinks of it before merging.

### CODE REVIEW

**Comment by robinMcA** [comment]
https://sonarcloud.io/dashboard?id=robinMcA_confluence-plugin-idea-search

File: src/main/java/au/com/agiledigital/idea_search/blueprints/KeyProperty.java [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 public class KeyProperty {
 
-  public String key;
-  public Object value;
-  public Options options;
+  private String key;
+  private Object value;
+  private Options options;
```
Comment (SamMurphyDev) [inline src/main/java/au/com/agiledigital/idea_search/blueprints/KeyProperty.java:7]: From memory, Velocity doesn't play nicely when these variables are private

File: src/main/java/au/com/agiledigital/idea_search/blueprints/Options.java [hunk 2]
```diff
@@ -2,10 +2,15 @@
...
-  public boolean isTechnology = false;
+  private boolean isDefault = false;
+  private boolean isUser = false;
+  private boolean isStatus = false;
+  private boolean isTechnology = false;
+
+  public boolean getIsDefault() {return isDefault;}
+  public boolean getIsUser() {return isUser;}
+  public boolean getIsStatus() {return isStatus;}
+  public boolean getIsTechnology() {return isTechnology;}
```
Comment (SamMurphyDev) [inline src/main/java/au/com/agiledigital/idea_search/blueprints/Options.java:13]: Velocity templates again

File: src/main/java/au/com/agiledigital/idea_search/macros/transport/IdeaContainer.java [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
+  private String blueprintId;
 
-  public MacroRepresentation technologies;
-  public MacroRepresentation description;
-  public MacroRepresentation status;
-  public MacroRepresentation owner;
+  private MacroRepresentation technologies;
+  private MacroRepresentation description;
+  private MacroRepresentation status;
+  private MacroRepresentation owner;
```
Comment (SamMurphyDev) [inline src/main/java/au/com/agiledigital/idea_search/macros/transport/IdeaContainer.java:19]: Velocity again

Comment (robinMcA) [reply src/main/java/au/com/agiledigital/idea_search/macros/transport/IdeaContainer.java]: I think that i have this working with the get...() methods.

File: src/main/java/au/com/agiledigital/idea_search/blueprints/Options.java (hunk 2 above)
Comment (robinMcA) [reply src/main/java/au/com/agiledigital/idea_search/blueprints/Options.java:13]: I think that i have this working with the get...() methods.

File: src/main/java/au/com/agiledigital/idea_search/blueprints/KeyProperty.java (hunk 1 above)
Comment (robinMcA) [reply src/main/java/au/com/agiledigital/idea_search/blueprints/KeyProperty.java:7]: I think that i have this working with the get...() methods.

**Review by SamMurphyDev — APPROVED** [review]
LGTM

**Pull request merged**